store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/AccumuloStoreConstants.java

```diff
@@ -69,8 +67,6 @@
     public static final int COLUMN_QUALIFIER_AGGREGATOR_ITERATOR_PRIORITY = 36;
     // Applied only during scans.
     public static final int ELEMENT_POST_AGGREGATION_FILTER_ITERATOR_PRIORITY = 37;
-    // Applied only during scans.
-    public static final int TRANSFORM_PRIORITY = 50;
```

Comment:
This constant isn't used but it is useful as you can see the order the iterators are applied.